data_acquisition/data_acquisition.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import os
+import psycopg2
+import getpass
+import logging
+
+# set up logging
+logging.basicConfig(format='%(asctime)s - %(levelname)s @' +
+                    '%(module)s-%(funcName)s : %(message)s',
+                    level=logging.INFO)
+
+# for connecting to remote database
+# conn = psycopg2.connect(database='yourdb', user='dbuser', password='abcd1234', host='server', port='5432', sslmode='require')
+# for creating local database initially
+# con = psycopg2.connect(dbname='postgres', user=getpass.getuser(), host='localhost', password='Pa55word')
+# cur = con.cursor()
+# cur.execute("CREATE TABLE Jobs(Id SERIAL PRIMARY KEY, job_title TEXT, link TEXT, description TEXT, city TEXT, state TEXT, country TEXT, latitude FLOAT, longitude FLOAT, company_name TEXT, salary_min INTEGER, start_date DATE)")
+# con.commit()
+# cur.close()
+# con.close()
+
+def request_ziprecruiter_jobs(page_num, api_key):
```

Comment:
It might be prudent to put a [docstring](https://www.python.org/dev/peps/pep-0257/) on the function. I know Pycharm can do them "automatically" (if you type three quotation marks), though I'm not sure if VS Code can as well.